website/src/theme/DocSidebar/index.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
Hi @bzp2010 it seems that this file has been  rewritten, would you please provide more details about this? thanks a lot

Reply:
I don't know more information. Previous versions of this file have this header text.

Reply:
not for the header text, but for the whole file, Is it because docusaurus has upgrade the template itself?

Reply:
The reason is that this template used JavaScript before, but now it is implemented by TypeScript. After that, the remaining templates will be migrated to TypeScript.